interop-future/jvm/src/main/scala/scalaz/zio/interop/Future.scala

```diff
@@ -0,0 +1,209 @@
+package scalaz.zio
+
+import scalaz.zio.internal.impls.Env
+import scala.concurrent.ExecutionContext
+import scala.util.{ Failure, Success, Try }
+import scala.reflect.ClassTag
+
+package object future {
+
+  /**
+   * A [[scala.concurrent.Future]] is a running computation, and corresponds
+   * most closely to ZIO `Fiber`.
+   */
+  type Future[+A] = Fiber[Throwable, A]
```

Comment:
could we define `Future`as:
```scala
ExecutionContext => Fiber[Throwable, A]
```
?

Reply:
Yes, but then we have 2 execution context in almost every method—which do we use? Although sometimes this would work better, and sometimes `IO` would work better, I think mapping `Future` to `Fiber` directly results in the "closest" alignment of concepts. It's not perfect though.